yarn.lock

```diff
@@ -9085,12 +9085,19 @@ ovh-ui-kit-bs@~1.3.x:
   resolved "https://registry.yarnpkg.com/ovh-ui-kit-bs/-/ovh-ui-kit-bs-1.3.3.tgz#31ed881f463b49fc91d418f7a9316e06e648d5b2"
   integrity sha512-H0r14QuDikarYlVqEpZIIM/9WroGqjzy9Z6l5hhchtATDl0kco3qa38ZpNK0hfUWr/s6LKHmREBd24AvxBljSw==
 
-ovh-ui-kit@^2.20.0, ovh-ui-kit@^2.21.0, "ovh-ui-kit@git+https://github.com/ovh-ux/ovh-ui-kit.git":
+ovh-ui-kit@^2.20.0, "ovh-ui-kit@git+https://github.com/ovh-ux/ovh-ui-kit.git":
   version "2.21.0"
   resolved "git+https://github.com/ovh-ux/ovh-ui-kit.git#ab2f4113de091bffe54fc11ffaa556fce94db3d3"
   dependencies:
     less-plugin-remcalc "^0.1.0"
 
+ovh-ui-kit@^2.21.0:
+  version "2.21.5"
```

Comment:
Is there a specific need here to upgrade `ovh-ui-kit`? I mean what was your requirements about this upgrade:)

Reply:
Yes it's needed for the underline header(and fix for the header-tabs)